From 93ed9f2f5d4b9e67c7097dbae4189a3110c78903 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Wed, 21 Jun 2017 20:05:40 +0800
Subject: [PATCH] Revert "usb: xhci-plat: check hcc_params after add hcd"

This reverts commit 4301c33db2829da207884f4c66bba798062db033.

Since we have merged upstream patch, so let's revert this patch.

Change-Id: I188629f566e50810bd633bff8cc63edda42efe28
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 drivers/usb/host/xhci-plat.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/usb/host/xhci-plat.c b/drivers/usb/host/xhci-plat.c
index f2173cc6ec1c..008def456f0a 100644
--- a/drivers/usb/host/xhci-plat.c
+++ b/drivers/usb/host/xhci-plat.c
@@ -210,9 +210,6 @@ static int xhci_plat_probe(struct platform_device *pdev)
 	if (ret)
 		goto dealloc_usb2_hcd;
 
-	if (HCC_MAX_PSA(xhci->hcc_params) >= 4)
-		xhci->shared_hcd->can_do_streams = 1;
-
 	return 0;
 
 
-- 
2.35.3

